docs/examples/custom_operations/custom_operator/naive_histogram/naive_histogram.h

```diff
@@ -0,0 +1,76 @@
+// Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef DALI_NAIVE_HISTOGRAM_H
+#define DALI_NAIVE_HISTOGRAM_H
+
+#include <vector>
+#include "dali/core/tensor_shape.h"
+#include "dali/pipeline/operator/operator.h"
+
+namespace naive_histogram {
+
+template<typename Backend>
+class NaiveHistogram : public ::dali::Operator<Backend> {
+ public:
+  explicit NaiveHistogram(const ::dali::OpSpec &spec) :
+          ::dali::Operator<Backend>(spec),
+          n_histogram_bins_(spec.GetArgument<int>("n_bins")) {}
```

Comment:
Maybe highlight this:
```suggestion
          // We obtain the values of scalar arguments from the OpSpec
          n_histogram_bins_(spec.GetArgument<int>("n_bins")) {}
```